Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml

```diff
@@ -18,69 +18,53 @@ args:
   name: brands
   auto: PREDEFINED
   predefined:
-  - 'SailPointIdentityNow'
   - 'Active Directory Query v2'
-  - 'PingOne'
   - 'Okta v2'
   - 'AWS - IAM'
   - 'Microsoft Graph User'
-  - 'SailPointIdentityIQ'
   - 'Cortex XDR - IR'
   - 'Cortex Core - IR'
-  - 'iam-get-user'
+  - 'PrismaCloud v2'
+  - 'AzureRiskyUsers'
 - description: Whether to retrieve human readable entry for every command or only the final result. True means to retrieve human readable entry for every command. False means to human readable only for the final result.
```

Comment:
Whether to fetch a human-readable entry for each command or just the final result. If set to true, all commands will have human-readable entries. If set to false, only the final result will be human-readable.